packages/dataviews/src/components/dataviews/stories/index.story.tsx

```diff
@@ -45,6 +45,19 @@ const defaultLayouts = {
 	[ LAYOUT_LIST ]: {},
 };
 
+function CustomLinkComponent( { item, ...props }: { item: SpaceObject } ) {
+	return (
+		<a
+			href="#"
+			onClick={ () => {
+				// eslint-disable-next-line no-alert
+				alert( 'Clicked: ' + item.title );
+			} }
```

Comment:
`onItemClick` included an automatic `stopPropagation` call, so now all click handlers need to include their own to prevent the row from being selected.

```suggestion
			onClick={ ( e ) => {
				e.stopPropagation();
				// eslint-disable-next-line no-alert
				alert( 'Clicked: ' + item.title );
			} }
```